kernelci/scheduler.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, configs, runtimes):
 
     def get_configs(self, event, channel='node'):
         """Get the scheduler configs matching a given event"""
+        # if event type is str, quit
+        if isinstance(event, str):
+            print("Event is a string! This is wrong, fix me")
+            return
```

Comment:
PubSub event payload is expected to be `dict`. A generalized solution will be:
```suggestion
        if not isinstance(event, dict):
            print("Error: Event payload must be a dictionary")
            return
```

Reply:
updated